R/parameters.R

```diff
@@ -517,3 +517,23 @@ parameterise_total_M <- function(parameters, total_M) {
   parameters$mosquito_limit <- ceiling(max_total_M * 5) #Allow for random fluctuations
   parameters
 }
+
+#' Use parameter draw from the join posterior
+#' 
+#' Overrides default (median) model parameters with a single draw from the fitted
+#' joint posterior. Must be called prior to set_equilibrium.
+#'
+#' @param parameters the model parameters
+#' @param draw the draw to use. Must be in integer between 1 and 1000
```

Comment:
"... must be an integer between ..."

Reply:
Thanks @htopazian 🙏,

Have fixed the typo.
Regarding the R/sysdata.rda file I just followed the steps [here](https://r-pkgs.org/data.html#sec-data-sysdata) for internal data in a package, so I'm not sure I can rename it?

Agree about the vignette! I was going to ask the vignette update squad to have a go at it  - I can add details to an issue to make sure it doesn't get forgotten

Reply:
Regarding dur_A, in the original params files I pulled out it is 195 - so I expect that the PMI paper SI documentation is just a mistake/rounding

Reply:
Just seen this, I didn't know the `data` directory had public visibility which would be a drawback...

Reply:
Just updated my comment for this

Reply:
Wait for dur_A, the default in malariasim is 200 https://github.com/mrc-ide/malariasimulation/blob/master/R/parameters.R#L240, but I think this should be 195 to match the original params files and the PMI documentation (I think the SI is correct).

Reply:
Ah, I set it to line up with the [legacy model](https://github.com/mrc-ide/Malaria_model/blob/master/input_files/model_parms.txt#L71)